src/main/java/com/ebay/myriad/scheduler/TaskFactory.java

```diff
@@ -63,6 +58,31 @@ private static String getFileName(String uri) {
             }
         }
 
+        private CommandInfo getCommandInfo() {
+            MyriadExecutorConfiguration myriadExecutorConfiguration = cfg.getMyriadExecutorConfiguration();
+            String cmdPrefix = "export CAPSULE_CACHE_DIR=`pwd`;echo $CAPSULE_CACHE_DIR; java -Dcapsule.log=verbose -jar ";
+            CommandInfo.Builder commandInfo = CommandInfo.newBuilder();
+            if (myriadExecutorConfiguration.getNodeManagerUri().isPresent() &&
+                    myriadExecutorConfiguration.getCommand().isPresent() &&
+                    myriadExecutorConfiguration.isRemoteDistribution()) {
+                String nmURI = myriadExecutorConfiguration.getNodeManagerUri().get();
+                URI executorURI = URI.newBuilder().setValue(nmURI).setExtract(true).build();
+                //@Todo: determine if "root" or "" is the proper default, to keep backward compatiblility
+                //keeping root for now
```

Comment:
The correct default here would be not to setUser, which tells the executor to use the FrameworkInfo.user, which itself could be root or "" (the user running the scheduler). In fact, let's remove this setUser statement altogether and just always use the FrameworkInfo.user, since no executors/tasks would need to run as any different user.